index.js

```diff
@@ -58,5 +58,6 @@ const main = (stream, options) => {
 };
 
 module.exports = main(process.stdout);
+module.exports.default = main(process.stdout);
```

Comment:
```suggestion
module.exports.default = module.export
```

For consistency and to ensure they're the same instance.

Reply:
Oh, right, overlooked this one.